### DISCUSSION TITLE
Admonitions - Callout implementation as part of core Astro or as an official module

### DISCUSSION THREAD

**Discussion by santm** [withastro/roadmap#825] [Proposal]
### Body

# Summary

Implementing a way to render Admonitions/Callout so people can use that without complex coding by themselves.  

# Background & Motivation

This is already part starlight theme as **\node_modules\@astrojs\starlight\integrations\asides.ts**

/**
 * remark plugin that converts blocks delimited with `:::` into styled
 * asides (a.k.a. “callouts”, “admonitions”, etc.). Depends on the
 * `remark-directive` module for the core parsing logic.
 *
 * For example, this Markdown
 *
 * ```md
 * :::tip[Did you know?]
 * Astro helps you build faster websites with “Islands Architecture”.
 * :::
 * ```
 *
 * will produce this output
 *
 * ```astro
 * <aside class="starlight-aside starlight-aside--tip" aria-label="Did you know?">
 *   <p class="starlight-aside__title" aria-hidden="true">Did you know?</p>
 *   <section class="starlight-aside__content">
 *     <p>Astro helps you build faster websites with “Islands Architecture”.</p>
 *   </section>
 * </Aside>
 * ```
 */

# Goals

A simple official plugin in coming days 

# Example

As shared on the above it is part of the starlight theme.

**Comment by stereobooster** [comment]
duplicate of https://github.com/withastro/roadmap/discussions/860
